content/html/concepts/tags/terms/span/span.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '<span>'
+Description: 'Used for grouping related text or elements for styling and scripting.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Structure'
+  - 'Style'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In HTML, the `<span>` tag is used for grouping text or elements together for styling and scripting. Unlike the block-level [`<div>` tag](https://www.codecademy.com/resources/docs/html/tags/div), `<span>` is an inline tag.
+
+## Syntax
+
+```html
+<span><!-- Place content here. --></span>
+```
+
+The `<span>` tag is usually wrapped inside another tag such as [`<li>`](https://www.codecademy.com/resources/docs/html/tags/li) or [`<p>`](https://www.codecademy.com/resources/docs/html/paragraphs). It can also use global attributes such as the following:
+
+- Selecting with [attributes](https://www.codecademy.com/resources/docs/html/attributes) such as [classes](https://www.codecademy.com/resources/docs/html/classes) for stylesheet rules or DOM manipulation.
+- Inline styling with the `style` attribute.
+- Creating custom property values with the `data-*` attribute.
+
```

Comment:
My only comment: Wouldn't this list and it's introductory sentence be better above the Syntax header? This is more of general usage, not syntax.